bin/reth/Cargo.toml

```diff
@@ -87,6 +87,7 @@ humantime = "2.1.0"
 
 [features]
 jemalloc = ["dep:jemallocator"]
+jemalloc-prof = ["jemalloc", "jemallocator?/profiling"]
```

Comment:
having this as separate feature makes sense

Reply:
ohhhh I was trying to figure out how to do this since using the profiling feature last, and didn't know about the `?`